api/src/main/java/org/openmrs/module/fhir/api/diagnosticreport/DiagnosticReportHandler.java

```diff
@@ -1,12 +1,19 @@
 package org.openmrs.module.fhir.api.diagnosticreport;
 
 import ca.uhn.fhir.model.dstu2.resource.DiagnosticReport;
+import org.openmrs.Obs;
+
+import java.util.List;
+import java.util.Set;
 
 public interface DiagnosticReportHandler {
-	
-	public DiagnosticReport generateFHIRDiagnosticReport(DiagnosticReportTemplate omrsDiagnosticReport);
-	
-	public DiagnosticReportTemplate generateOpenMRSDiagnosticReport(DiagnosticReport fhirDiagnosticReport);
-	
+
 	public String getId();
+
+	public Set<Obs> saveObs(List<Obs> result);
```

Comment:
why we have save obs here. We can do this in OpenMRS API. So why we are having separate methods in here?